src/main/java/fi/aalto/cs/apluscourses/intellij/model/SettingsImporter.java

```diff
@@ -67,6 +68,22 @@ public void importIdeSettings(@NotNull Course course) throws IOException {
     PluginSettings.getInstance().setImportedIdeSettingsId(course.getId());
   }
 
+  /**
+   * Imports the VM options from the course configuration file into the IDE. If there are no
+   * options to import, this function does nothing.
+   */
+  @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+  public void importVMOptions(@NotNull Course course) {
+    if (!VMOptions.canWriteOptions()) {
+      return;
```

Comment:
Notification or at least a line in the log?

Reply:
Will do